### PR TITLE
Regenerated NRP to include latest features

### PR DESCRIPTION
**Pull request by EvgenyAgafonchikov**

### CODE REVIEW

**Comment by DeepakRajendranMsft** [comment]
@balajikris Could you please take a look at this PR and merge, so @EvgenyAgafonchikov can send the PR the cli asap

File: lib/services/networkManagement2/package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "Rajendran, Deepak <derajen@microsoft.com>",
     "Choubey, Ankur <anchoube@microsoft.com>"
   ],
-  "version": "1.3.0-preview",
+  "version": "1.3.1-preview",
```
Comment (balajikris) [inline lib/services/networkManagement2/package.json]: you're changing the API version but you're only updating the patch version for the package?

**Comment by balajikris** [comment]
This should be closed in favor of #2219. Reopen if thats not the case.

**Pull request closed without merging**